tests/test_mockgun.py

```diff
@@ -42,19 +42,15 @@
 from shotgun_api3 import ShotgunError
 
 
-def setUpModule():
-    """
-    Sets the Shotgun schema for this series of unit tests.
-    """
-    mockgun_schema_folder = os.path.join(
-        os.path.dirname(__file__),
-        "mockgun"
-    )
+mockgun_schema_folder = os.path.join(
```

Comment:
Python 2.5's unit tests module doesn't support the setUpModule method, so do it globally instead.